src/main/java/com/javabrains/PracticeREST/UserEntity.java

```diff
@@ -7,11 +7,16 @@ public static void main(String[] args) {
         boolean isTrue;
 
         if(1>0){
-            isTrue=false;
+            isTrue=true;
         }
 
         String dependsOnIsTrue = isTrue ? "nice" :"bad ";
-        System.out.println(dependsOnIsTrue);
 
+
+    }
+
+    void helloWorld(){
+
```

Comment:
Nilesh add access modifier public tto improve readability

Reply:
Added public access modifier for the corresponding method